.github/workflows/ohos.yml

```diff
@@ -53,7 +60,7 @@ jobs:
       - name: Setup OpenHarmony SDK
         id: setup_sdk
         # FIXME: We pin the exact patch version to the LKG version of the
-        # action to address bugs in the most recent 0.1.2 release. Once that
+        # action to address bugs in the most recent 0.1.3 release. Once that
         # has been addresssed upstream, we can revert to specifying only the
         # major.minor tag.
         uses: openharmony-rs/setup-ohos-sdk@v0.1.2
```

Comment:
I'm curious about this change, because 0.1.2 is still indicated below.

Reply:
Sorry for the confusion. The original comment I had added was incorrect - 0.1.2 is the last known good version and 0.1.3 is the buggy one, while the comment claimed it was 0.1.2 that has the bug. The change in this PR is just correcting the comment to reflect that.